### PR TITLE
IMoD v1.5 Update

### PR DESCRIPTION
**Pull request by intika**
- new feature : min fake dialog
- bug fix WebView loop
- bug fix blanc menu entry

### CODE REVIEW

File: app/src/main/java/de/Maxr1998/xposed/maxlock/ui/settings/GuideFragment.java [hunk 1]
```diff
@@ -41,7 +43,10 @@ public void onReceivedError(WebView view, int errorCode, String description, Str
                 if (index != -1) {
                     url = url.substring(0, index);
                 }
-                webView.loadUrl(url);
```
Comment (intika) [inline app/src/main/java/de/Maxr1998/xposed/maxlock/ui/settings/GuideFragment.java]: Bug fix (loop)

File: app/src/main/java/de/Maxr1998/xposed/maxlock/ui/settings/GuideFragment.java [hunk 2]
```diff
@@ -41,7 +43,10 @@ public void onReceivedError(WebView view, int errorCode, String description, Str
...
+                if (avoidLoopCounter > 0) {
+                    webView.loadUrl(url);
+                    avoidLoopCounter = avoidLoopCounter-1;
+                }
             }
 
             @Override
```
Comment (intika) [inline app/src/main/java/de/Maxr1998/xposed/maxlock/ui/settings/GuideFragment.java]: Bug fix loop

File: app/src/main/res/menu/toolbar_menu.xml [hunk 3]
```diff
@@ -9,6 +9,7 @@
     <item
         android:id="@+id/toolbar_master_switch"
         android:title=""
+        android:visible="false"
         app:actionViewClass="android.support.v7.widget.SwitchCompat"
         app:showAsAction="always" />
 </menu>
```
Comment (intika) [inline app/src/main/res/menu/toolbar_menu.xml]: Bug fix (empty menu)

Comment (Maxr1998) [inline app/src/main/res/menu/toolbar_menu.xml]: I don't understand this bug..

**Pull request merged**